engine/engine-common.gyp

```diff
@@ -30,6 +30,7 @@
 					[
 						'<@(perl)',
 						'../util/encode_version.pl',
+						'-<(git_revision)',
```

Comment:
@livecodefraser points out he needed a similar workaround in places. eg:
```
# Really nasty hack to prevent this from being treated as a path
'$(this_is_an_undefined_variable)MCstartupstack',
```
Im my case I actually need the `-` anyway but if we want to be consistent then it's easy enough to use the above.